pages/blog/_.vue

```diff
@@ -24,6 +24,18 @@
           v-if="page.keywords"
           :tags="page.keywords"
         />
+
+        <div
+          v-if="enableBlogComments"
```

Comment:
Get rid of the 'ENABLE_BLOG_COMMENTS' variable from the .env file
Instead have this be conditional on DISQUS_SHORTNAME not being undefined, that can be loaded into a variable where ` enableBlogComments()` is now. Then just check that variable here.